packages/react/src/components/organization-switcher/organization-switcher.test.tsx

```diff
@@ -139,6 +139,38 @@ describe("OrganizationSwitcher", () => {
     expect(currentValue).toBe(org.org_name);
   });
 
+  test("should display the current organizations label override as the selected value", async () => {
+    const currentOrg = createTestOrganization();
+    const user = createTestUser({ oid: currentOrg.id });
+    const anotherOrg = createTestOrganization();
+    const spy = vi.fn(async () => [anotherOrg, currentOrg]);
+
+    const expected = faker.commerce.product();
+
+    user.getOrganizations = spy;
+
+    const { container } = render(
+      <TestSlashIDProvider user={user}>
+        <OrganizationSwitcher
+          renderLabel={(org) => {
+            if (org.id === currentOrg.id) return expected;
+            return org.org_name;
+          }}
+        />
+      </TestSlashIDProvider>
+    );
+
+    await waitFor(() =>
+      expect(screen.queryByText("-")).not.toBeInTheDocument()
```

Comment:
You're right but I couldn't think of another way to do it, because the loading state is actually the same component as the rendered state - there are no differences.

I think testing for the current value is probably a better idea, let me try

Reply:
Done! It's a bit noisy but more future proof